subprojects/language-java/src/integTest/groovy/org/gradle/api/tasks/compile/AggregatingIncrementalAnnotationProcessingIntegrationTest.groovy

```diff
@@ -146,7 +145,7 @@ class AggregatingIncrementalAnnotationProcessingIntegrationTest extends Abstract
         run "compileJava"
 
         then:
-        outputs.recompiledFiles("AHelper", "B", "BHelper", "ServiceRegistry", "ServiceRegistryResource.txt")
```

Comment:
This change and the one below are the important parts - They ensure that the minimal set of classes is reprocessed by Micronaut.